src/app.md

```diff
@@ -725,13 +726,63 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-command]: #ex-semver-command
+<a name="ex-semver-command"></a>
+## Check external command version for compatibility
+[![semver-badge]][semver] [![cat-text-processing-badge]][cat-text-processing] [![cat-os-badge]][cat-os]
+
+Runs `git --version` using [`Command`], then parses the version number into a [`semver::Version`]
+using [`Version::parse`]. A [`semver::VersionReq`] is used to compare the parsed version to a 
+minimum version requirement.
+
+```rust,no_run
+# #[macro_use]
+# extern crate error_chain;
+extern crate semver;
+
+use std::process::Command;
+use semver::{Version, VersionReq};
+#
+# error_chain! {
+#     foreign_links {
+#         Io(std::io::Error);
+#         Utf8(std::string::FromUtf8Error);
+#         SemVer(semver::SemVerError);
+#         SemVerReq(semver::ReqParseError);
+#     }
+# }
+
+fn run() -> Result<()> {
+    let version_test = VersionReq::parse("> 1.12.0")?;
+    let output = Command::new("git").arg("--version").output()?;
+
+    if !output.status.success() {
+        bail!("Command executed with failing error code");
+    }
+
+    let stdout = String::from_utf8(output.stdout)?;
+    // `git --version` output: "git version x.y.z"
+    let version = stdout.split(" ").last().expect("Invalid command output");
+    let parsed_version = Version::parse(version)?;
+
+    if !version_test.matches(&parsed_version) {
+        bail!("Command version lower than minimum supported version")
```

Comment:
how about adding the parsed version number to the `bail!` invocation? 
`bail!` has format capabilities identical to the `println!`